vars/runPipeline.groovy

```diff
@@ -387,6 +387,7 @@ def call() { timeout(time: 1, unit: 'HOURS') {
         "dlang-community/D-YAML", // 15s
         "dlang-community/libdparse", // 13s
         "dlang-community/dfmt", // 11s
+        "dlang/tools", // not known yet
```

Comment:
It takes quite a while at the moment - 2m44s:

![image](https://user-images.githubusercontent.com/4370550/35329630-dff7b992-0100-11e8-8f82-f4e0783864a3.png)

However, this is due to using digger which builds `dmd`, `druntime` and `phobos` from scratch. Once this PR is merged, we don't need to do this as the tools's testsuite will be run on every PR to dmd, druntime and tools. So we can remove the entire digger setup at `tools` and then update the time. I suspect it's somewhere between 30-60s.

Reply:
tools PR: https://github.com/dlang/tools/pull/298